doc/dev/internals.rst

```diff
@@ -2,10 +2,14 @@
  Ceph Internals
 ================
 
-.. note:: If you're looking for how to use Ceph as a library from your
-   own software, please see :doc:`/api/index`.
+.. note:: For information on how to use Ceph as a library (from your own
```

Comment:
As a .... library?  Is this talking about librados?

Reply:
I'm not sure. This was introduced on 26 Jul 2018, long before my time. I figured I'd clean up the sentence before we started in on the technical details.

Reply:
ack.